docs/page/configuration.md

```diff
@@ -39,9 +39,10 @@ settings.
     will automatically discover if `dap-mode` is present and it will
     download and install the required server side components. If you
     have already downloaded a `Eclispe JDT Server` you will have to
-    force a server update via `lsp-java-update-server`. In order to
-    enable lsp java, you will have to require `dap-java.el`.
-    
+    force a server update via `C-u M-x lsp-install-server` and select
```

Comment:
```suggestion
    force a server update via <kbd>C-u</kbd> <kbd>M-x</kbd> `lsp-install-server` and select
```

Reply:
Just tried the new "batch" feature from github. If you want a regular commit and push, let me know.